### ISSUE TITLE
Where can I get DPDK driver?

### ISSUE THREAD

**Issue by hokim72** [Xilinx/open-nic#3]
FAQ says

### What SW support can I expect to use, for example, DPDK, eBPF, VM, etc.?
Xilinx provides the linux open-nic-driver sources, and  **an excellent DPDK driver has been developed by the community**.

Where can I get it ?
 [DPDK](https://github.com/Xilinx/dma_ip_drivers/tree/master/QDMA/DPDK) can be used for open-nic?

**Comment by cneely-xlnx** [comment]
Yes, DPDK can be used for open-nic, although we have only tested it using DPDK's pktgen.  The version of the DPDK driver based is for DPDK v20.11.  This driver was ported from the same basic Xilinx QDMA DPDK drivers, which you linked to/mentioned in your question.  The drivers were ported by someone in the community and they achieve very high performance.  We haven't posted this code yet (modifications relative to the QDMA drivers), but we plan to do this.  I'll plan to post an update to this message soon with more information.

**Comment by hokim72** [comment]
Thanks for your reply

**Comment by cneely-xlnx** [comment]
> Thanks for your reply

If you are interested in evaluating the preliminary version of the DPDK driver and providing us with some feedback, please let me know.  Please email me directly at cneely@xilinx.com.

**Comment by cneely-xlnx** [comment]
For info, the DPDK driver-related code and instructions have now been posted to:
https://github.com/Xilinx/open-nic-dpdk
